internal/pkg/helm/values_inspector.go

```diff
@@ -91,3 +100,141 @@ func isBooleanYamlValue(input *map[string]interface{}, path *[]string, pathIndex
 		}
 	}
 }
+
+//GetValues .. Get values  fro given path
```

Comment:
Typo.  Additionally, this comment really doesn't explain what this method is trying to do.  Can you please elaborate on the description.

Reply:
Added more explanations to the comment